cmd/queue/main.go

```diff
@@ -133,26 +134,49 @@ var (
 		stats.UnitMilliseconds)
 )
 
-func initEnv() {
-	containerConcurrency = util.MustParseIntEnvOrFatal("CONTAINER_CONCURRENCY", logger)
-	queueServingPort = util.MustParseIntEnvOrFatal("QUEUE_SERVING_PORT", logger)
-	revisionTimeoutSeconds = util.MustParseIntEnvOrFatal("REVISION_TIMEOUT_SECONDS", logger)
-	servingConfig = util.GetRequiredEnvOrFatal("SERVING_CONFIGURATION", logger)
-	servingNamespace = util.GetRequiredEnvOrFatal("SERVING_NAMESPACE", logger)
-	servingPodIP = util.GetRequiredEnvOrFatal("SERVING_POD_IP", logger)
-	servingPodName = util.GetRequiredEnvOrFatal("SERVING_POD", logger)
-	servingRevision = util.GetRequiredEnvOrFatal("SERVING_REVISION", logger)
-	servingService = os.Getenv("SERVING_SERVICE") // KService is optional
-	userTargetPort = util.MustParseIntEnvOrFatal("USER_PORT", logger)
+type envConfig struct {
+	ContainerConcurrency   string `envconfig:"CONTAINER_CONCURRENCY" required:"true"`
+	QueueServingPort       string `envconfig:"QUEUE_SERVING_PORT" required:"true"`
+	RevisionTimeoutSeconds string `envconfig:"REVISION_TIMEOUT_SECONDS" required:"true"`
+	ServingConfiguration   string `envconfig:"SERVING_CONFIGURATION" required:"true"`
+	ServingNamespace       string `envconfig:"SERVING_NAMESPACE" required:"true"`
+	ServingPodIP           string `envconfig:"SERVING_POD_IP" required:"true"`
+	ServingPod             string `envconfig:"SERVING_POD" required:"true"`
+	ServingRevision        string `envconfig:"SERVING_REVISION" required:"true"`
+	// optional
```

Comment:
maybe add `//optional` behind the field's definition to have the formatter align it all.

Reply:
Modified